src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/SolutionExplorer/ProjectNodeCommandHandler.cs

```diff
@@ -100,7 +100,8 @@ private void OpenFile(WorkspaceVisualNodeBase node)
                 {
                     Assumes.NotNull(openDocumentService);
```

Comment:
Remove this, as we're no longer assuming `openDocumentService` is not null.

Reply:
Removed, thanks to both! @drewnoakes